### PR TITLE
fix: postcss responsive variations

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description

Updated Dialtone postCSS responsive variations plugin.

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [ ] Update, remove, or extend all affected documentation.
 - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -30,17 +30,18 @@ const breakpoints = [
...
-  /^\.d-mx([0-9]*|-(auto|unset))$/m, // Margin X
-  /^\.d-g-cols[0-9]*$/m, // Grid columns
-  /^\.d-(stack|flow|h|w|fs-)[0-9]*$/m, // Stack, Flow, Height, Widths and Font sizes
+  /\.d-mx([0-9]*|-(auto|unset))$/, // Margin X
+  /\.d-g-cols[0-9]*$/, // Grid columns
+  /\.d-(stack|flow|h|w|fs-)[0-9]*$/, // Stack, Flow, Height, Widths and Font sizes
   '.d-w100p',
-  /^\.d-wmx(-(auto|unset)|[0-9]*(ch|p))$/m, // Max widths
+  /\.d-wmx(-(auto|unset)|[0-9]*(ch|p))$/, // Max widths
+  '.d-bgc-red-200',
```
Comment (braddialpad) [inline gulpfile.js]: This shouldn't be here right? Why would we convert only 1 color?

Comment (juliodialpad) [reply gulpfile.js]: Ah, yeah, I used that for testing. Will remove it as we don't need it.

**Comment by braddialpad** [comment]
These ones in the gulpfile are just for the docsite right? it has nothing to do with our published package?

**Comment by juliodialpad** [comment]
> These ones in the gulpfile are just for the docsite right? it has nothing to do with our published package?

Right, it's the local config to our docsite, they're not included in the package.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**